kpm/lib/kpm/system.rb

```diff
@@ -209,8 +211,21 @@ def show_plugin_information(bundles_dir, output_as_json)
       all_plugins
     end
 
+    def get_kaui_standalone_version(kaui_web_path = nil)
+      kaui_search_default_dir = Dir[kaui_web_path.nil? ? '' : kaui_web_path][0] || DEFAULT_KAUI_SEARCH_BASE_DIR
+      version = nil
```

Comment:
Does it make sense to default to `''` when `Dir[kaui_web_path.nil?` is true? I know this is the same paradigm used for retrieving `kaui` and `killbill` version. Shouldn't we use instead `DEFAULT_KAUI_SEARCH_BASE_DIR ` ?